atom/browser/atom_browser_context.cc

```diff
@@ -78,10 +78,10 @@ AtomBrowserContext::AtomBrowserContext(const std::string& partition,
   base::StringToInt(command_line->GetSwitchValueASCII(switches::kDiskCacheSize),
                     &max_cache_size_);
 
-  if (!base::PathService::Get(DIR_USER_DATA, &path_)) {
-    base::PathService::Get(DIR_APP_DATA, &path_);
+  if (!base::PathService::Get(DIR_USER_CACHE, &path_)) {
+    base::PathService::Get(DIR_CACHE, &path_);
     path_ = path_.Append(base::FilePath::FromUTF8Unsafe(GetApplicationName()));
-    base::PathService::Override(DIR_USER_DATA, path_);
+    base::PathService::Override(DIR_USER_CACHE, path_);
```

Comment:
If we're setting `DIR_USER_CACHE` we probably also want to ensure it exists, as we do for `DIR_USER_DATA` in `Browser::DidFinishLaunching()`?